packages/loot-core/src/server/db/mutations/accounts.ts

```
@@ -0,0 +1,56 @@
+    'SELECT * FROM accounts WHERE offbudget = ? ORDER BY sort_order, name',
+    [account.offbudget ? 1 : 0],
+  );
+
+  // Don't pass a target in, it will default to appending at the end
+  const { sort_order } = shoveSortOrders(accounts);
+
+  account = accountModel.validate({ ...account, sort_order });
+  return insertWithUUID('accounts', account);
+}
+
+export function updateAccount(account: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)